packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -238,7 +242,6 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
                                             autoComplete='off'
                                             placeholder='Enter your document number'
                                             value={values.document_number}
-                                            onPaste={e => e.preventDefault()}
```

Comment:
Should we not assign any function to onPaste prop to handle the functionality?

Reply:
I'm not sure, what functionality we should handle. Function here just did preventDefault. 
I didn’t find any proper similar cases in the code. Any suggestions?

Reply:
How do you handle it if user enters the example number in the field?

Reply:
I'm checking it on lines 96-98

Reply:
@matin-deriv The example numbers that were originally ment were the sequential and repetitive numbers in error messages (for example 123456789 and 111111111). These numbers get already blocked by logic I implemented, the user will get an error message when blurring the input.